configs/callbacks/early_stopping.yaml

```diff
@@ -0,0 +1,15 @@
+# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.EarlyStopping.html#pytorch_lightning.callbacks.EarlyStopping
```

Comment:
I think we can shorten these links?
```suggestion
# https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.EarlyStopping.html
```

Reply:
Yes, that will be more clear, i will change this and add some comment to every argument